stdlib/socketserver.pyi

```diff
@@ -153,7 +154,7 @@ class StreamRequestHandler(BaseRequestHandler):
     wbufsize: ClassVar[int]  # undocumented
     timeout: ClassVar[float | None]  # undocumented
     disable_nagle_algorithm: ClassVar[bool]  # undocumented
-    connection: _socket  # undocumented
+    connection: SSLSocket  # undocumented
```

Comment:
In the code, I see `self.connection = self.request`, and we type `BaseRequestHandler.request` as Any. What guarantees that the socket is always an SSLSocket?

Reply:
maybe `type[_socket]` then?

Reply:
I'm pretty sure it's an instance, not a class. Perhaps we should just use Any here too.

Reply:
I really should look into preparing an `AnyOf` PEP, despite the resistance. There are so many instances where this would be useful.

Reply:
Very happy to give it a proof-read (or help co-write) if you ever do :)